### ISSUE TITLE
Nice start

### ISSUE THREAD

**Issue by johncoleman83** [Tuxsi-curry/hello_world#2]
Hey! Nice job getting on github. I hope you can keep learning and working on some coding projects. Let me know if you need any guidance or suggestions on how to keep learning. Or if you get stuck on a code / or design challenge let me know.

- This is a guide of a lot of the things that helped me become a professional software engineer: https://johncoleman83.github.io/software_engineering_hub/
  - here is the readme markdown file that is the base for that html document: 
https://github.com/johncoleman83/software_engineering_hub

- Here’s a blog of how I broke into the industry: https://davidjohncoleman.com/2018/self-status-reinvented/
  - I like sharing and helping others because my career has been very wonderful and fulfilling and since I was able to break into the industry as an old 30+ year old, I think any of you young people with all your learning potential can definitely do it.

**Comment by johncoleman83** [comment]
Hey No activity in 20 days?  Have you been able to start coding?

**Comment by Curry4-ui** [comment]
Yes, thanks to you. I have been reading different articles and haven’t
managed to get on git hub.

**Comment by johncoleman83** [comment]
Oh okay sounds good. LMK if you need anything.

**Comment by Curry4-ui** [comment]
Thank you, I will
